package.json

```diff
@@ -46,10 +45,10 @@
     "supertest": "^3.3.0"
   },
   "dependencies": {
+    "@babel/core": "^7.1.2",
+    "@babel/node": "^7.0.0",
+    "@babel/preset-env": "^7.1.0",
```

Comment:
These are dev dependencies

Reply:
Ah ok, I thought that we needed them for the simulado binary...